public/stylesheets/dashboard.css

```diff
@@ -0,0 +1,19 @@
+.cc-leftPanel {
+  height: 85vh;
+  border-right: 1px solid #F0F0F0;
+}
+
+.cc-userIcon {
```

Comment:
You may want to think about being more specific with this selector. What if, for instance, you needed to display the user's icon elsewhere but as a square, or in a different size?